docs_src/design/TOC.md

```diff
@@ -1,12 +1,19 @@
 # Use Cases and Design Records
 
 ## Use Case Records (UCRs)
+
+!!! note 
+    UCRs are listed in alphabetical order by title. New UCRs should be inserted in the proper order.
+
 | Name/Link                                                                 | Short Description                                             |
 |---------------------------------------------------------------------------|---------------------------------------------------------------|
-| [0001 System Events for Devices](./ucr/0001-System-Events-for-Devices.md) | Use Case for System Events for Device add/update/delete |
+| [System Events for Devices](./ucr/System-Events-for-Devices.md) | Use Case for System Events for Device add/update/delete |
 
 ## Architectural Design Records (ADRs)
 
+!!! note
+    ADRs are listed in chronological order. New ADRs should prepend the next sequence number to the title and be added at the bottom list.
```

Comment:
```suggestion
    ADRs are listed in chronological order. New ADRs should prepend the next sequence number to the title and be added at the bottom of the list.
```

Reply:
Actually, I feel the second sentence which is instructions for the author, doesn't need to be in the note and appear on docs. I suggest including it as a comment. Same for the UCR note.

Reply:
> and appear on docs

No clear what you mean 

> I suggest including it as a comment.

Comment where?

Reply:
Reworked the notes.

Reply:
I meant comments in markdown with `<!-- my comment -->` HTML syntax.